plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -3079,10 +3107,13 @@ public void registerCoreMembers() {
         //
         // @Description
         // TODO: CUBOID OPTION
-        // TODO: Document Command Details
+        // Allows removal of an entity or list of entities.
+        // If used on a npc, it will remove the npc fully, just like using (/npc remove)
+        // This command can remove any entity, including a player's entity as well.

```

Comment:
Should mention the result of specify an entity type (rather than specific spawned entity) as the input

Reply:
And also mention what removing a player does